lib/resources/dstu2/medication_statement.yaml

```diff
@@ -0,0 +1,259 @@
+                 "coding": [
+                   {"system": "http://snomed.info/sct",
+                    "code": "26643006",
+                    "display": "Oral route (qualifier value)",
+                    "userSelected": false}],
+                 "text": "Oral"},
+               "quantityQuantity": {
+                 "value": 325,
+                 "units": "mg",
+                 "system": "http://unitsofmeasure.org",
+                 "code": "mg"}}]}
+  children:
+
+  - name: text
+    required: 'No'
+    cardinality: 0..1
+    type: string
+    description: Free text dosage information as reported about a patient's medication use.
+    example: '{"text": "2, Oral, 2x/Wk, first dose 05/05/15 10:00:00 EDT"}'
+
+  - name: timing
+    required: 'No'
+    cardinality: 0..1
+    type: Timing
+    description: The timing schedule for giving the medication to the patient.
+    example: |
+               {"timing": {
+                 "repeat": {
+                   "boundsPeriod": {
+                     "start": "2014-11-03T14:38:00.000-05:00"}},
+                 "code": {
+                   "coding": [
+                     "system": "http://hl7.org/fhir/v3/vs/GTSAbbreviation",
+                     "code": "TID",
+                     "display": "TID"],
+                   "text": "TID"}}
+
+  - name: asNeeded[x]
+    required: 'No'
+    cardinality: 0..1
+    type: boolean|CodeableConcept
+    description: Whether the Medication is only taken when needed within a specific dosing schedule.
+    example: |
+               asNeededCodeableConcept
+                 {"asNeededCodeableConcept": {
+                   "coding": [
+                     {"system": "http://snomed.info/sct",
+                      "code": "25064002",
+                      "display": "for headache"}]}}
+
+               asNeededBoolean
+                 {"asNeededBoolean" : true}
+    url: http://hl7.org/fhir/DSTU2/medicationstatement-definitions.html#MedicationStatement.dosage.asNeeded_x_
+
+  - name: route
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description: >
+                     A code specifying the route or physiological path of administration of a therapeutic
+                     agent into or onto a subject.
+    example: |
+              {"route": {
+                "coding": [
+                  {"system": "http://snomed.info/sct",
+                   "code": "41974700",
+                   "display": "Chew - dosing instruction imperative (qualifier value)"}],
+                "text": "Chewed"}}
+    binding:
+      description: >
+                      A coded concept describing the route or physiological path of administration of a
+                      therapeutic agent into or onto the body of a subject.
+      terminology:
+      - display: SNOMED CT
+        system: http://snomed.info/sct
+
+  - name: quantity[x]
+    required: 'No'
+    cardinality: 0..1
+    type: SimpleQuantity|Range
```

Comment:
Range and SimpleQuantity need to be added to the types.yaml here: https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/dstu2/types.yaml

They are not linking since they aren't defined